__tests__/app.test.js

```diff
@@ -146,6 +147,35 @@ describe('/api/articles/:article_id', () => {
     })
 })
 
+describe('/api/articles', () => {
+    test('GET 200: responds with an array of objects of articles, each with the correct properties', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body: {articles}}) => {
+            articles.forEach((article) => {
+                expect(typeof article.author).toBe('string')
+                expect(typeof article.title).toBe('string')
+                expect(typeof article.article_id).toBe('number')
+                expect(typeof article.topic).toBe('string')
+                expect(typeof article.created_at).toBe('string')
+                expect(typeof article.votes).toBe('number')
+                expect(typeof article.article_img_url).toBe('string')
+                expect(typeof article.comment_count).toBe('string')
```

Comment:
we're also expecting it to not have a body property!

Reply:
would that just be expect(typeof article.body).toBe('undefined') ?